### PR TITLE
Release/2.0

### PR DESCRIPTION
**Pull request by jasonleenaylor**
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20)

### CODE REVIEW

**Review by andrew-polk — APPROVED** [review]
Approving, but someone who knows more than me (aka anything) about the yml files should look.

> Reviewed 17 of 17 files at r1.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20#-:-MXi37P85NhJcQrBSXwb:b97pky3)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ermshiperete and @tombogle)

**Review by andrew-polk — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ermshiperete and @tombogle)

**Review by andrew-polk — APPROVED** [review]
> Reviewed 2 of 2 files at r3.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ermshiperete and @tombogle)

**Review by andrew-polk — APPROVED** [review]
> Reviewed 3 of 3 files at r4.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ermshiperete and @tombogle)

**Review by jasonleenaylor — COMMENTED** [review]
Thanks @andrew-polk I had to do some tweaking to get the build action to pass on windows (there is a bug in github actions that has nuget.org missing from the package sources) I'm eager to try and tag the repo once this is merged to see if the deploy.yml will automatically build and push the package to Nuget.org

> *[Reviewable](https://reviewable.io/reviews/sillsdev/desktopanalytics.net/20#-:-MXiDq_Q652w2FArWYqs:bqbqwme)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ermshiperete and @tombogle)

**Comment by jasonleenaylor** [comment]
@tombogle  Note that merging this will also close issue #18

File: src/DesktopAnalytics/Analytics.cs [hunk 1]
```diff
@@ -144,9 +144,11 @@ public Analytics(string apiSecret, UserInfo userInfo, Dictionary<string, string>
 			SetApplicationProperty("FullVersion", versionNumberWithBuild);
 			SetApplicationProperty("UserName", GetUserNameForEvent());
 			SetApplicationProperty("Browser", GetOperatingSystemLabel());
+			SetApplicationProperty("64bit OS", Environment.Is64BitOperatingSystem.ToString());
+			SetApplicationProperty("64bit App", Environment.Is64BitProcess.ToString());
 
 
-			if (string.IsNullOrEmpty(AnalyticsSettings.Default.LastVersionLaunched))
+            if (string.IsNullOrEmpty(AnalyticsSettings.Default.LastVersionLaunched))
```
[the remark this answers is not in the document]
Comment (jasonleenaylor) [reply src/DesktopAnalytics/Analytics.cs:151]: Hmm, nope.

File: src/DesktopAnalytics/DesktopAnalytics.csproj [hunk 2]
```diff
@@ -1,105 +1,45 @@
...
+<PropertyGroup>
+	<TargetFramework>net461</TargetFramework>
+	<PackageId>SIL.DesktopAnalytics</PackageId>
+	<Authors>John Hatton, Stephen McConnel, Tom Bogle, Eberhard Beilharz, John Thomson, Andrew Polk, Jason Naylor</Authors>
+	<Company>SIL International</Company>
+	<PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<PackageProjectUrl>https://github.com/sillsdev/desktopanalytics.net</PackageProjectUrl>
+	<RepositoryUrl>https://github.com/sillsdev/desktopanalytics.net</RepositoryUrl>
+	<Description>Wrapper around segment.com's Analytics.net specifically for desktop apps (instead of servers).</Description>
+</PropertyGroup>
```
[the remark this answers is not in the document]
Comment (jasonleenaylor) [reply src/DesktopAnalytics/DesktopAnalytics.csproj:12]: Added symbol package

**Pull request merged**